auth/lib/models/code.dart

```diff
@@ -113,18 +113,20 @@ class Code {
     String secret,
     CodeDisplay? display,
     int digits,
+    Algorithm? algorithm,
```

Comment:
These can be named parameters, we don't have to modify existing usage for these methods.